AUTHENTICATION.md

```diff
@@ -28,22 +22,25 @@ providing **Project ID** and **Service Account Credentials** directly in code.
 
 ### Google Cloud Platform environments
 
-While running on Google Cloud Platform environments such as Google Compute Engine, Google App Engine and
-Google Kubernetes Engine, no extra work is needed. The **Project ID** and **Credentials** and are 
-discovered automatically from the attached service account. Code should be written as if already authenticated.
+While running on Google Cloud Platform environments such as Google Compute Engine, Google App Engine
+and Google Kubernetes Engine, no extra work is needed. The **Project ID** and **Credentials** and
+are discovered automatically from the attached service account. Code should be written as if already
+authenticated.
 
 For more information, see
 [Set up ADC for Google Cloud services](https://cloud.google.com/docs/authentication/provide-credentials-adc#attached-sa).
 
 ### Environment Variables
 
-**NOTE**: This library uses [`getenv`](https://www.php.net/manual/en/function.getenv.php), so if your environemnt 
-variables are set in PHP, they must use [`putenv`](https://www.php.net/manual/en/function.putenv.php),
+**NOTE**: This library uses [`getenv`](https://www.php.net/manual/en/function.getenv.php), so if
+your environemnt variables are set in PHP, they must use
+[`putenv`](https://www.php.net/manual/en/function.putenv.php),
 
 ```php
 putenv("GOOGLE_APPLICATION_CREDENTIALS=" . __DIR__ . '/your-credentials-file.json');
 ```
-The **Project ID** and **Credentials JSON** can be placed in environment variables instead of declaring them directly in code.
+The **Project ID** and **Credentials JSON** can be placed in environment variables instead of
+declaring them directly in code.
```

Comment:
Similarly, the Project ID is not discovered via Env Var for GAPIC libraries, so we should remove this
```suggestion
The path to the **Credentials JSON** can be placed in environment variables instead of
declaring it directly in code.
```

That means we should also remove lines 45-49 below